### PR TITLE
Update for 0.5.0 release and include text on automatic injection: MAI…

### PR DESCRIPTION
**Pull request by knrc**
…STRA-131

### CODE REVIEW

**Comment by knrc** [comment]
@brian-avery @geekspertise Can you take a look at the content and provide feedback?

File: topics/docs/automatic-injection.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: "Installing the sidecar within your Service Mesh Applications."
+type: "document"
+category: "Getting started"
+description: "Service Mesh provides features to applications through the istio-proxy sidecar within your applications's pod, created through automatic injection of the configuration."
+---
+
+Istio relies on the existence of a proxy sidecar within the application's pod to provide the Service Mesh capabilities to the application.  The proxy sidecar can be included through a manual process, prior to deployment, however we recommend automatic injection as this ensures your application contains the appropriate configuration for your service mesh at the time of its deployment.
+
+Automatic Sidecar Injection
```
Comment (brian-avery) [inline topics/docs/automatic-injection.adoc]: Can we mention manual as well as automatic?

File: topics/docs/install.adoc [hunk 2]
```diff
@@ -13,8 +13,8 @@ Guide for installing Istio into an existing OpenShift Container Platform (OCP) o
 
 - OpenShift Container Platform (OCP) 3.11 (https://docs.openshift.com/container-platform/3.11/install/prerequisites.html)
 - Origin (OKD) 3.11
-- istiooc (https://github.com/Maistra/origin/releases/tag/v3.11.0%2Bmaistra-0.4.0)
-- NOTE: OpenShift Online and OpenShift Dedicated are not supported in the 0.4.0 Istio Tech Preview
+- istiooc (https://github.com/Maistra/origin/releases/tag/v3.11.0%2Bmaistra-0.5.0)
```
Comment (brian-avery) [inline topics/docs/install.adoc]: Can we fix the references to 0.3 in here?

Comment (knrc) [reply topics/docs/install.adoc]: Yes, I can add those.  We must have missed them in the 0.4 update.

File: topics/docs/automatic-injection.adoc (hunk 1 above)
Comment (knrc) [reply topics/docs/automatic-injection.adoc]: I've added a section on manual injection, let me know your thoughts

File: topics/docs/automatic-injection.adoc [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      - name: sleep
+        image: tutum/curl
+        command: ["/bin/sleep","infinity"]
+        imagePullPolicy: IfNotPresent
+----
+
+Manual Sidecar Injection
+------------------------
+
+Manual injection of the sidecar is supported through the upstream istioctl command.  To obtain the executable and deploy an application using manual injection you should
```
[the remark this answers is not in the document]
Comment (knrc) [reply topics/docs/automatic-injection.adoc]: This was included in @geekspertise's comments and is now outdated

**Comment by knrc** [comment]
@brian-avery Any objections to me merging this?

**Comment by brian-avery** [comment]
@knrc lgtm

**Pull request merged**